server/pulp/server/async/tasks.py

```diff
@@ -198,7 +198,7 @@ def from_task_status_dict(cls, task_status):
         :returns: a TaskResult containing the task in it's spawned_tasks lsit
         :rtype: TaskResult
         """
-        return cls(spawned_tasks=[{'task_id': task_status.get('task_id')}])
+        return cls(spawned_tasks=[{'task_id': task_status.task_id}])
```

Comment:
@asmacdo take a look at the traceback in the description of the bug https://pulp.plan.io/issues/320
TaskStatus does not have a 'get' attribute, but it does have 'task_id' attribute